src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-shadow */
+import React from 'react';
+import './ProductCard.scss';
+import { PhoneCard } from '../../types/PhoneCard';
+import phonePhoto from '../../phone-photo.png';
+
+const phoneCard = {
+  id: '1',
+  category: 'phones',
+  phoneId: 'apple-iphone-7-32gb-black',
+  itemId: 'apple-iphone-7-32gb-black',
+  name: 'Apple iPhone 7 32GB Black',
+  fullPrice: 400,
+  price: 375,
+  screen: '4.7 IPS',
+  capacity: '32GB',
+  color: 'black',
+  ram: '2GB',
+  year: 2016,
+  image: 'img/phones/apple-iphone-7/black/00.jpg',
+};
+
+export const ProductCard: React.FC<PhoneCard> = ({ phoneCard }) => {
+  const {
+    name,
+    fullPrice,
+    price,
+    screen,
+    capacity,
+    ram,
+  } = phoneCard;
+
+  return (
+    <div className="card">
+      <img
+        className="card__img"
+        src={phonePhoto}
+        alt="iPhone"
+        width="208px"
+        height="196px"
+      />
+
+      <h2 className="card__title">
+        {name}
+      </h2>
+
+      <p className="card__price">
+        <span className="card__price-current">
+          {`$${price}`}
```

Comment:
```suggestion
          ${price}
```
I guess you can write it just like this) 
the `$` here is like a part of JSX so like a regular text and then inside  `{}` you use some variable)